### DISCUSSION TITLE
[Feature Request]: 90 degree rotation of display on Heltec Wireless Paper

### DISCUSSION THREAD

**Discussion by CTLS** [meshtastic/firmware#5379] [Ideas]
### Platform

ESP32

### Description

It would be really nice to be able to rotate the screen 90 degrees on the Heltec Wireless Paper for compact housing designs.

**Comment by todd-herbert** [comment]
I've got something in progress for this one, but it's still in the proof of concept stage; no guarantees or timeframes just yet.
